### PR TITLE
add monster ai 01,2,3,4,6

### PR DESCRIPTION
**Pull request by Iambananayeah**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: Silksong/Assets/Scripts/MapObjects/AttackTest/PlayerAttack.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ void Start()
     // Update is called once per frame
     void Update()
     {
-        if(Input.GetKeyDown(KeyCode.C))//������
```
Comment (jelly-konjac) [inline Silksong/Assets/Scripts/MapObjects/AttackTest/PlayerAttack.cs:26]: 既然Bullet class你删了，那我感觉这部分也不用comment了，直接删了会比较好

**Review by jelly-konjac — APPROVED** [review]

File: Silksong/Assets/Scripts/MapObjects/AttackTest/PlayerAttack.cs (hunk 1 above)
Comment (Iambananayeah) [reply Silksong/Assets/Scripts/MapObjects/AttackTest/PlayerAttack.cs:26]: 我把bullet class改成bulletCollider class了，用于子弹的碰撞销毁等。 这个playerAttack暂时测试用，到时候是要删掉的。

File: Silksong/Assets/Scripts/Kimo_FSM/FSM_abstract/FSMManager.cs [hunk 2]
```diff
@@ -11,12 +11,15 @@ public abstract class FSMManager<T1,T2> : MonoBehaviour
 {
 
     public Animator animator;
-    public AudioSource audio;
-    public Rigidbody2D rigidbody;
+    public AudioSource audios;
+    public Rigidbody2D rigidbody2d;
+    public bool FaceLeftFirstOriginal;//ԭͼ�Ƿ�����
```
Comment (IsaacJade) [inline Silksong/Assets/Scripts/Kimo_FSM/FSM_abstract/FSMManager.cs:16]: 这些涉及怪物组件的代码为什么要写在FSMManager这个基类里？

**Review by IsaacJade — APPROVED** [review]
Pass

File: Silksong/Assets/Scripts/Kimo_FSM/FSM_abstract/FSMManager.cs (hunk 2 above)
Comment (Iambananayeah) [reply Silksong/Assets/Scripts/Kimo_FSM/FSM_abstract/FSMManager.cs:16]: 为了方便引用这些组件

**Pull request merged**